src/main/java/com/onlinebookshop/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.onlinebookshop.service.impl;
+
+import com.onlinebookshop.model.Book;
+import com.onlinebookshop.repository.BookRepository;
+import com.onlinebookshop.service.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+    @Autowired
+    private BookRepository bookRepository;
```

Comment:
In most cases it's considered better approach to use constructor injection (there are some exceptions though)